python_modules/dagster/dagster/_core/definitions/op_invocation.py

```diff
@@ -158,7 +159,10 @@ def op_invocation_result(
         from dagster._config.pythonic_config import Config
 
         context = (context or build_op_context()).replace_config(
-            config_input._as_config_dict()  # noqa: SLF001
+            config_dictionary_from_values(
+                config_input._as_config_dict(),  # noqa: SLF001
+                config_input.to_config_schema().as_field(),
+            )
```

Comment:
similar to feedback on other diff. should this not live in `_as_config_dict`?

Reply:
stacked on #13876 and updated